File: app/src/main/java/com/cn29/aac/binding/EditTextBindingUtil.java
package com.cn29.aac.binding;

import android.widget.EditText;

import androidx.databinding.InverseMethod;

/**
 * Created by Charles Ng on 7/9/2017.
 */

public class EditTextBindingUtil {
  @InverseMethod("inverseText")
  public String bindText(EditText editText, String newValue) {
    editText.setText(newValue);
    return newValue;
  }

  public String inverseText(EditText editText) {
    return editText.getText().toString();
  }
}

File: app/src/main/java/com/cn29/aac/binding/EditTextBindingUtil.kt
package com.cn29.aac.binding

import android.widget.EditText
import androidx.databinding.InverseMethod

/**
 * Created by Charles Ng on 7/9/2017.
 */
class EditTextBindingUtil {
    @InverseMethod("inverseText")
    fun bindText(editText: EditText,
                 newValue: String): String {
        editText.setText(newValue)
        return newValue
    }

    fun inverseText(editText: EditText): String {
        return editText.text.toString()
    }
}